argocd/overlays/moc-infra/projects/apicurio-registry.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
 spec:
   destinations:
     - namespace: 'apicurio-*'
-      server: 'smaug'
+      server: 'https://api.smaug.na.operate-first.cloud:6443'
```

Comment:
whoops apologies I missed this last time, for `server` you need to have the url, but if you want to use the _name_ then you have to use the `destination.name` field, like [here](https://github.com/operate-first/apps/blob/master/argocd/overlays/moc-infra/applications/envs/moc/smaug/b4mad/racing.yml#L7)

Reply:
np, it's fine now

Reply:
> it's fine now 

Can you switch this to `name` instead? Like so:
```yaml
  destinations:
    - namespace: 'apicurio-*'
      name: 'smaug'
```